### ISSUE TITLE
Steelseries Sensei 310 no longer recognized

### ISSUE THREAD

**Issue by Yasand123** [libratbag/libratbag#1467]
**Information**
- `ratbagd` version (`ratbagd --version`): 0.17
- Distribution: ArchLinux
- Kernel version (ex. `uname -srmo`): `Linux 6.2.7-arch1-1 x86_64 GNU/Linux`

**Describe the bug**
libratbag no longer recognizes my Steelseries Sensei 310. This only started happening recently. Originally my mouse was working just fine, was recognized out of the box.

**To Reproduce**
1. Open Piper, with sudo or without, or type `ratbagctl list`.
2. Says no devices available.

**Logs**
```
$ ratbagd --verbose=raw
Initializing libratbag
ratbag debug: New device: Cooler Master Technology Inc. AMD SR4 lamplight Control
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: No data file found for 2516:0051
ratbag debug: New device: Cooler Master Technology Inc. AMD SR4 lamplight Control
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: No data file found for 2516:0051
ratbag debug: New device: Cooler Master Technology Inc. AMD SR4 lamplight Control
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: No data file found for 2516:0051
ratbag debug: New device: AsusTek Computer Inc. AURA LED Controller
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: No data file found for 0b05:18f3
ratbag debug: New device: CASUE USB KB
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: No data file found for 2a7a:9587
ratbag debug: New device: CASUE USB KB
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: No data file found for 2a7a:9587
ratbag debug: New device: Burr-Brown from TI               USB Audio CODEC 
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: No data file found for 08bb:2902
ratbag debug: New device: SteelSeries SteelSeries Sensei 310 eSports Mouse
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: device assigned driver steelseries
ratbag debug: hidraw info: bus 0x03 vendor 0x1038 product 0x1722
ratbag debug: SteelSeries SteelSeries Sensei 310 eSports Mouse is device '/dev/hidraw7'.
ratbag debug: Parsing HID report descriptor
ratbag debug: Parsing HID report descriptor
ratbag debug: hidraw info: bus 0x03 vendor 0x1038 product 0x1722
ratbag debug: SteelSeries SteelSeries Sensei 310 eSports Mouse is device '/dev/hidraw8'.
ratbag debug: Parsing HID report descriptor
ratbag debug: Parsing HID report descriptor
ratbag debug: hidraw info: bus 0x03 vendor 0x1038 product 0x1722
ratbag debug: SteelSeries SteelSeries Sensei 310 eSports Mouse is device '/dev/hidraw9'.
ratbag debug: Parsing HID report descriptor
ratbag debug: Parsing HID report descriptor
ratbag debug: - HID report ID 01
ratbag debug: - HID report ID 02
ratbag debug: New device: SteelSeries SteelSeries Sensei 310 eSports Mouse
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: device assigned driver steelseries
ratbag debug: hidraw info: bus 0x03 vendor 0x1038 product 0x1722
ratbag debug: SteelSeries SteelSeries Sensei 310 eSports Mouse is device '/dev/hidraw7'.
ratbag debug: Parsing HID report descriptor
ratbag debug: Parsing HID report descriptor
ratbag debug: hidraw info: bus 0x03 vendor 0x1038 product 0x1722
ratbag debug: SteelSeries SteelSeries Sensei 310 eSports Mouse is device '/dev/hidraw8'.
ratbag debug: Parsing HID report descriptor
ratbag debug: Parsing HID report descriptor
ratbag debug: hidraw info: bus 0x03 vendor 0x1038 product 0x1722
ratbag debug: SteelSeries SteelSeries Sensei 310 eSports Mouse is device '/dev/hidraw9'.
ratbag debug: Parsing HID report descriptor
ratbag debug: Parsing HID report descriptor
ratbag debug: - HID report ID 01
ratbag debug: - HID report ID 02
ratbag debug: New device: SteelSeries SteelSeries Sensei 310 eSports Mouse
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: device assigned driver steelseries
ratbag debug: hidraw info: bus 0x03 vendor 0x1038 product 0x1722
ratbag debug: SteelSeries SteelSeries Sensei 310 eSports Mouse is device '/dev/hidraw7'.
ratbag debug: Parsing HID report descriptor
ratbag debug: Parsing HID report descriptor
ratbag debug: hidraw info: bus 0x03 vendor 0x1038 product 0x1722
ratbag debug: SteelSeries SteelSeries Sensei 310 eSports Mouse is device '/dev/hidraw8'.
ratbag debug: Parsing HID report descriptor
ratbag debug: Parsing HID report descriptor
ratbag debug: hidraw info: bus 0x03 vendor 0x1038 product 0x1722
ratbag debug: SteelSeries SteelSeries Sensei 310 eSports Mouse is device '/dev/hidraw9'.
ratbag debug: Parsing HID report descriptor
ratbag debug: Parsing HID report descriptor
ratbag debug: - HID report ID 01
ratbag debug: - HID report ID 02
DBus server ready
```

**Additional context**
As mentioned before. Initially this was a non issue, it started happening only recently. Sadly I cannot say when exactly. One big change I did was migrating from GNOME to KDE Plasma, not sure if this is relevant, but who knows.

**Comment by staticssleever668** [comment]
Hello, could you downgrade to libratbag v0.16 and tell me if the issue is fixed?
Probably the same issue as #1438

**Comment by Yasand123** [comment]
> Hello, could you downgrade to libratbag v0.16 and tell me if the issue is fixed? Probably the same issue as #1438

Downgrading to v0.16 fixed the issue. Seems to be the same issue then.
